src/server/worker/pipeline/transform/transform_test.go

```diff
@@ -110,7 +110,7 @@ func newWorkerSpawnerPair(t *testing.T, dbConfig serviceenv.ConfigOption, pipeli
 
 	eg.Go(func() error {
 		err := backoff.RetryUntilCancel(env.driver.PachClient().Ctx(), func() error {
-			return Worker(env.driver.PachClient().Ctx(), env.driver, env.logger, &Status{})
+			return Worker(env.driver.PachClient().Ctx(), env.driver, env.logger, &Status{}, "")
```

Comment:
this call should use the original Worker() signature

Reply:
good catch, fixed it in the latest commit